tests/vars/rh_distros_vars.yml

```diff
@@ -0,0 +1,19 @@
+# vars for handling conditionals for RedHat and clones
+# DO NOT EDIT - file is auto-generated
+---
+# Ansible distribution identifiers that the role treats like RHEL
+__network_rh_distros:
+  - AlmaLinux
+  - CentOS
+  - OracleLinux
+  - RedHat
+  - Rocky
+
+# Same as above but includes Fedora
+__network_rh_distros_fedora: "{{ __network_rh_distros + ['Fedora'] }}"
+
+# Use this in conditionals to check if distro is Red Hat or clone
+__network_is_rh_distro: "{{ ansible_distribution in __network_rh_distros }}"
+
+# Use this in conditionals to check if distro is Red Hat or clone, or Fedora
+__network_is_rh_distro_fedora: "{{ ansible_distribution in __network_rh_distros_fedora }}"
```

Comment:
I think that the tests will load the variables defined in `defaults/main.yml`, why we need a separate file for defining the variables specifically for tests?

Reply:
> I think that the tests will load the variables defined in `defaults/main.yml`,

No, they won't, not before the role is included.

> why do we need a separate file for defining the variables specifically for tests?

Many of the tests check the distribution and version before including the role.

Reply:
See the Q&A section in the updated description